stack.yaml

```diff
@@ -54,6 +54,7 @@ extra-deps:
 - data-clist-0.1.2.0
 - word-wrap-0.2
 - vty-5.18.1
+- concise-0.1.0.1
```

Comment:
this should get picked up in relevant lts builds very soon.  but yeah whatevs :)